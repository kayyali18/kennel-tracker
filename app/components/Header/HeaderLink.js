import { Link } from 'react-router-dom'
import styled from 'styled-components'

export default styled(Link)`
  display: inline-flex;
  padding: 0.25em 2em;
  margin: calc(10px + 6 * ((100vw - 320px) / 680));
  text-decoration: none;
  border-radius: 8px;
  -webkit-font-smoothing: antialiased;
  -webkit-touch-callout: none;
  user-select: none;
  cursor: pointer;
  outline: 0;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  font-weight: bold;
  font-size: calc(16px + 6 * ((100vw - 320px) / 680));
  border: none;
  color: #ff4a6e;

  &:active {
    background: red;
    color: #fff;
  }

  &:hover {
    background: linear-gradient(#ff700b, #ff275f);
    color: #fff;
  }
`
