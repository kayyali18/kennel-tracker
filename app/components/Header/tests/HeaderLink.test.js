import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import HeaderLink from '../HeaderLink'

describe('<HeaderLink />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = shallow(<HeaderLink />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
