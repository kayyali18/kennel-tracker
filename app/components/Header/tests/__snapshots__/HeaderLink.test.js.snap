// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderLink /> should match the snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "HeaderLink-sc-1mtyaiv-0",
        "isStatic": true,
        "rules": Array [
          "display:inline-flex;padding:0.25em 2em;margin:calc(10px + 6 * ((100vw - 320px) / 680));text-decoration:none;border-radius:8px;-webkit-font-smoothing:antialiased;-webkit-touch-callout:none;user-select:none;cursor:pointer;outline:0;font-family:'Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:bold;font-size:calc(16px + 6 * ((100vw - 320px) / 680));border:none;color:#ff4a6e;&:active{background:red;color:#fff;}&:hover{background:linear-gradient(#ff700b,#ff275f);color:#fff;}",
        ],
      },
      "displayName": "HeaderLink",
      "render": [Function],
      "styledComponentId": "HeaderLink-sc-1mtyaiv-0",
      "target": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    }
  }
  forwardedRef={null}
/>
`;
